readme.md

```diff
@@ -74,6 +74,10 @@ This might be useful if you want to teardown the queue at the end of your progra
 
 Note: This does not cancel promises that are already running.
 
+### limit.wait()
```

Comment:
Not sure about `.wait()` for the name. Is it really clear enough at the callsite what it does? "It waits for what?".